src/core/src/bootstrap/EnvLayer.py

```diff
@@ -396,4 +401,15 @@ def utc_to_standard_datetime(utc_datetime):
         def standard_datetime_to_utc(std_datetime):
             """ Converts datetime object to string of format '"%Y-%m-%dT%H:%M:%SZ"' """
             return std_datetime.strftime("%Y-%m-%dT%H:%M:%SZ")
+
+        @staticmethod
+        def datetime_string_to_posix_time(datetime_string, format_string):
+            """ Converts string of given format to posix datetime string.
+                type: (str, str) -> str"""
```

Comment:
Excluding calling context as this is in a base layer - conformant posix time is int, not str

Reply:
Done